### PR TITLE
Binary Tree zigzag traversal Added

### PR DESCRIPTION
**Pull request by Devforlife07**
Fixes #

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- 
- 

#### Files Added:
- 
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: treezigzagtraversal/zigzag.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import java.util.*;
+
+/**
+ * zigzag
+ */
+// sample input
+// 10 true 20 true 40 false false true 50 false false true 30 true 60 false
+// false true 73 false false
+// sample output
+// 10 30 20 40 50 60 73
```
Comment (SakshiShreya) [inline treezigzagtraversal/zigzag.java]: Either use single line comments or multi line comment here.
Also, sample input and output should be at the EOF.

File: treezigzagtraversal/zigzag.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+        }
+    }
+
+    public static void main(String[] args) {
+        root = buildtree();
+        zigzag();
+    }
+
+}
```
Comment (SakshiShreya) [inline treezigzagtraversal/zigzag.java]: Add a new line at the EOF

**Comment by raghav-dalmia** [comment]
Add the issue number in the description. Also, squash your commits.

**Comment by somya-kapoor** [comment]
@Devforlife07 Are you working on it?

**Comment by singh-shreya6** [comment]
No changes made from 25 days. Closing it. If you want to work on it again. Please reach out.

**Pull request closed without merging**